doc/nrf/libraries/bluetooth_services/services/hogp.rst

```diff
@@ -7,50 +7,38 @@ GATT Human Interface Device (HID) Service Client
    :local:
    :depth: 2
 
-The HID Service Client module can be used to interact with a connected HID server.
-The server can, for example, be an instance of the :ref:`hids_readme`.
-
-The HID Service Client is used in the :ref:`bluetooth_central_hids` sample.
+The HID Service Client module is used to interact with a connected HID server.
 
+Overview
+********
 
 The client uses the :ref:`gatt_dm_readme` module to acquire all attribute handles that are required to interact with the HID server.
+
 Some additional data must be read from the discovered descriptors before the HID client is ready.
 This process is started automatically just after the handles are assigned.
 If the process finishes successfully, the :c:type:`bt_hogp_ready_cb` function is called.
 Otherwise, :c:type:`bt_hogp_prep_fail_cb` is called.
 
-
 Configuration
 *************
 
-Apart from standard configuration parameters, there is one important setting:
-
-:kconfig:option:`CONFIG_BT_HOGP_REPORTS_MAX`
-  Sets the maximum number of total reports supported by the library.
-  The report memory is shared along all HIDS client objects, so this option should be set to the maximum total number of reports supported by the application.
+Apart from standard configuration parameters, you can use the :kconfig:option:`CONFIG_BT_HOGP_REPORTS_MAX` setting that sets the maximum number of total reports supported by the library.
+The report memory is shared along all HIDS client objects, so this option should be set to the maximum total number of reports supported by the application.
 
 Usage
 *****
 
-.. note::
-   Do not access any of the values in the :c:struct:`bt_hogp` object structure directly.
-   All values that should be accessed have accessor functions.
-   The reason that the structure is fully defined is to allow the application to allocate the memory for it.
-
+You can use the GATT HID Client module to execute actions described in the sections below.
```

Comment:
In general, it is not a good practice to refer to something below or above. Maybe something like "subsequent". Or just try go figure out what might work. (I have already ignored a few earlier in this review.)